maintenance/scripts/setup.sh

```diff
@@ -1,5 +1,30 @@
 #! /bin/bash
 
+Help()
+{
+  echo "Setup the project and your environment."
+  echo
+  echo "Syntax: setup.sh [-h|c]"
+  echo "   -h   Show this help message."
+  echo "   -c   Setup the environment for CI."
+  echo
+}
+
+CI=false
+
+while getopts ":hc" option; do
+  case $option in
+    h | help) # display Help
```

Comment:
I put this through shellcheck and i don't think there's a long-form option for getopts -- maybe you're mixing up conventions for shell's getopts vs perl getopt?
https://stackoverflow.com/questions/402377/using-getopts-to-process-long-and-short-command-line-options

Reply:
I'm a little confused by the comment 😕 This is just allowing people to spell help out if they prefer to be more verbose

```sh
docker-compose run --rm maintenance bash setup.sh -help
```